qemu/tests/cfg/blockdev_inc_backup_xpt_incon_bitmap.cfg

```diff
@@ -0,0 +1,32 @@
+- blockdev_inc_backup_xpt_incon_bitmap:
+    only Linux
+    only filesystem
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_inc_backup_xpt_incon_bitmap
+    virt_test_type = qemu
+    images += " data"
+    source_images = "data"
+    image_backup_chain_data = "base"
+    remove_image_data = yes
+    force_create_image_data = yes
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data = 2G
+    image_size_base = 2G
+
+    image_format_data = qcow2
+    image_format_base = qcow2
+
+    image_name_data = data
+    image_name_base = base
+    rebase_mode = unsafe
+    storage_type_default = "directory"
+
+    nbd_export_format_data = qcow2
+    nbd_port_data = 10850
+    export_cmd = "%s -f ${nbd_export_format_data} -p ${nbd_port_data} -t -B %s %s/${image_name_data}.${image_format_data}"
```

Comment:
let's add '--fork' in the qemu-nbd command line, or if qemu-nbd succeeds, process.run will not return.

Reply:
Have added --fork --pid-file /tmp/qemu_nbd_pid.